src/aro/target.zig

```diff
@@ -723,6 +724,168 @@ pub fn toLLVMTriple(target: std.Target, buf: []u8) []const u8 {
     return stream.getWritten();
 }
 
+fn isPIEDefault(d: *const Driver) bool {
```

Comment:
These functions shouldn't depend on `Driver`; to make them more generalizable I would suggest that we have it return an enum with values `yes`, `no`, or `depends_on_linker`. And then resolve that to a bool in the caller where we know what the linker is.